webapp/tests/Unit/Controller/API/ProblemControllerTest.php

```diff
@@ -47,4 +49,20 @@ class ProblemControllerTest extends BaseTest
     ];
 
     protected array $expectedAbsent = ['4242', 'nonexistent'];
+
+    public function testDeleteNotAllowed(): void
+    {
+        if ($this->apiUser) {
```

Comment:
Probably more robust to check if the apiUser is not admin?

Reply:
Done, also added a 401 test for add